src/ddl/fluview.sql

```diff
@@ -269,6 +269,7 @@ CREATE TABLE `fluview_clinical` (
   `percent_a` double DEFAULT NULL,
   `percent_b` double DEFAULT NULL,
   PRIMARY KEY (`id`),
+  UNIQUE KEY `release_date_2` (`release_date`, `epiweek`, `region`),
   KEY `release_date` (`release_date`),
```

Comment:
```suggestion
  UNIQUE KEY `release_date` (`release_date`, `epiweek`, `region`),
```

you can get rid of that other `release_date` key, the new one will be used in its place (see https://dev.mysql.com/doc/refman/8.0/en/multiple-column-indexes.html )